handler/example_test.go

```diff
@@ -41,7 +41,7 @@ func ExampleNewPause() {
 		os.Exit(1)
 	}
 
-	c, err := controller.NewUnmanaged("pod", mgr, controller.Options{
+	c, err := controller.NewUnmanaged("pod", controller.Options{
```

Comment:
It seems right since the mgr option was removed here: https://github.com/kubernetes-sigs/controller-runtime/pull/3141